africa_capitals.txt

```diff
@@ -0,0 +1,55 @@
+algeria algiers no +1 africa 
+angola luanda no +1 africa 
+benin porto-novo no +1 africa 
+botswana gaborone no +2 africa 
+burkina_faso ouagadougou no +0 africa 
+burundi gitega no +2 africa 
+chad n'djamena no +1 africa 
+democratic_republic_of_the_congo kinshasa no +1 africa 
+djibouti djibouti no +3 africa 
+egypt cairo no +2 africa 
+eritrea asmara no +3 africa 
+ethiopia addis_ababa no +3 africa 
+gabon libreville no +1 africa 
+gambia banjul no +0 africa 
+ghana accra no +0 africa 
+guinea conakry no +0 africa 
+guinea_bissau bissau no +0 africa 
+equatorial_guinea malabo no +1 africa 
+cameroon yaoundé no +1 africa 
+kenya nairobi no +3 africa 
+chambers moroni no +3 africa 
+congo brazzaville no +1 africa 
+lesotho maseru no +2 africa 
+liberia monrovia no +0 africa 
+libya tripoli no +2 africa 
+madagascar antananarivo no +3 africa 
+malawi lilongwe no +2 africa 
+mali bamako no +0 africa 
+morocco rabat +0 +1 africa 
+mauritania nouakchott no +0 africa 
+mauritius port_louis no +4 africa 
+mozambique maputo no +2 africa 
+namibia windhoek no +2 africa 
+niger niamey no +1 africa 
+nigeria abuja no +1 africa 
+south_africa pretoria no +2 africa 
+central_african_republic bangui no +1 africa 
+cape_verde praia no -1 africa 
+rwanda kigali no +2 africa 
+western_sahara al-ujun +0 +1 africa 
```

Comment:
Po angielsku się to pisze inaczej

Reply:
Tylko ostatnie zmieniłem, a reszta po sprawdzeniu na angielskiej wiki wygląda OK.
Zmieniłem al-ujun na el-aaiun

>Nazwa Al-Ujun to polska transkrypcja arabskiej nazwy miasta. W języku hiszpańskim nazwa miasta brzmi El Aaiún (używana była przed zajęciem terytorium przez Maroko). W marokańskiej literaturze używana jest raczej francuska nazwa Laâyoune, która nie jest jednak akceptowana przez rdzennych mieszkańców tych ziem.

https://en.wikipedia.org/wiki/Bangui
https://en.wikipedia.org/wiki/Praia
https://en.wikipedia.org/wiki/Kigali

Jeśli jednak to też nie do końca dobrze, to daj znać i jak możesz to podpowiedz.